assets/terraform-modules/bootkube/variables.tf

```
@@ -173,3 +173,15 @@ variable "enable_tls_bootstrap" {
   description = "Enable TLS Bootstrap for Kubelet."
   type        = bool
 }
+
+variable "failsafe_inbound_host_ports" {
+  description = "UDP/TCP/SCTP protocol/port pairs that Felix will allow incoming traffic to host endpoints on irrespective of the security policy."
```

```suggestion
  description = "UDP/TCP/SCTP protocol/port pairs to allow incoming traffic on regardless of the security policy."
```

- Avoid the future tense.
- Simplify the phrasing and make it less ambiguous.

Also, what format is expected here? `tcp/123` for example?

---

Format here is 

```yaml
- protocol: tcp
  port: 22
```